Source/JavaScriptCore/API/tests/testapi.c

```diff
@@ -1578,12 +1578,15 @@ int main(int argc, char* argv[])
     configureJSCForTesting();
 
 #if !OS(WINDOWS)
-    char resolvedPath[PATH_MAX];
-    if (!realpath(argv[0], resolvedPath))
+    char *resolvedPath = realpath(argv[0], NULL);
+    if (!resolvedPath)
         fprintf(stdout, "Could not get the absolute pathname for: %s\n", argv[0]);
-    char* newCWD = dirname(resolvedPath);
-    if (chdir(newCWD))
-        fprintf(stdout, "Could not chdir to: %s\n", newCWD);
+    else {
+        char *newCWD = dirname(resolvedPath);
+        if (chdir(newCWD))
+            fprintf(stdout, "Could not chdir to: %s\n", newCWD);
```

Comment:
Might as well opportunistically switch to stderr :

```suggestion
            fprintf(stderr, "Could not chdir to: %s\n", newCWD);
```

Reply:
There's only one other use of `fprintf(stdout` in this file and it's two lines up, so might as well fix them both here.